lib/pdf_fill/extras_generator_v2.rb

```diff
@@ -5,6 +5,68 @@ class ExtrasGeneratorV2 < ExtrasGenerator
     HEADER_FONT_SIZE = 14.5
     SUBHEADER_FONT_SIZE = 10.5
 
+    class Question
```

Comment:
I decided to make Ruby classes to more clearly organize the question-centric code, but the main functionality here is the same -- `render_question` turned into `Question#render`.